Popover docs (#1884)
telerik/blazor-docs: components/popover/animation.md

---
title: Animation
page_title: Popover Animation
description: Explore the animation settings of the Popover for Blazor. Discover how to adjust the way the Popover appears and disappears on the screen.
slug: popover-animation
tags: telerik,blazor,popover,animation
published: True
position: 35
---

# Popover Animation Settings

This article outlines the available settings that allow you to customize the animations when the Popover displays and hides.

## Type

You can change the way the Popover component flows in and out of the screen by setting the `AnimationType` parameter to a member of the `AnimationType` enum:

* `None`
* `Fade`
* `PushUp`
* `PushDown`
* `PushLeft`
* `PushRight`
* `RevealVertical`
* `SlideUp`
* `SlideIn`
* `SlideDown` (default)
* `SlideRight`
* `SlideLeft`
* `ZoomIn`
* `ZoomOut`

See the [example](#example) below to customize the available parameters and see how they affect the Popover component.

## Duration

Set the `AnimationDuration` parameter in milliseconds as `int` to control how long the animation will take until the component is fully displayed.

> See the [example](#example) below to customize the available parameters and see how they affect the Popover component.

Reviewer: ```suggestion
The [example](#example) below lets you customize the available `AnimationDuration` parameter and see how it affects the Popover component.
```
